etc/BUILD

```diff
@@ -0,0 +1,9 @@
+py_binary(
+    name = "file_to_string",
+    srcs = [
+        "file_to_string.py",
+    ],
+    main = "file_to_string.py",
+    visibility = ["//visibility:public"],
+    # deps = ["@openroad-pip//matplotlib"],
```

Comment:
this looks like a leftover comment.

Reply:
It is. I thought a pip install something would be required.

Reply:
file_to_string doesn't really have a `mathplotlib` dependency. Looking at the Python file, I see `base64`, `sys`, `os` and `argsparse`.
There is also a superfluous `json` dependency in the Python file, but that looks like a leftover that can be removed.

Reply:
none of these need be pip installed